app/src/main/java/org/thoughtcrime/securesms/trustedIntroductions/glue/AttachmentTableGlue.kt

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2024 Signal Messenger, LLC
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+package org.thoughtcrime.securesms.trustedIntroductions.glue
+
+import androidx.annotation.NonNull
+import org.thoughtcrime.securesms.attachments.Attachment
+import org.thoughtcrime.securesms.trustedIntroductions.TI_Utils
+import java.io.ByteArrayOutputStream
+import java.io.InputStream
+
+object AttachmentTableGlue {
+
+
+  /**
+   * TODO
+   *
+   * @param attachment the attachment to be evaluated
+   * @return a ByteinputStream containing the contents of the attachment
+   */
+  @JvmStatic
+  fun grabIntroductionData(attachment: Attachment, inputStream: InputStream): InputStream {
+    var text = ""
```

Comment:
Why you removed type? I think it was better

Reply:
Because my IDE was marking it as superfluous :)